CHANGELOG.md

```diff
@@ -74,7 +74,7 @@ Now shows an error:
 
 ```console
 $ node example.js a b c
-error: too many arguments. Expected 0 arguments but got 3.
+error: too many arguments. Expected 0 arguments but got 3: a, b, c.
```

Comment:
This is the description of the behaviour in Commander v13, which has not changed.

The PR should not be updating the CHANGELOG.

Reply:
Indeed, I was doing too quickly didn't look at the file name. I also realised that implementation is doing completely wrong thins since it compares argument name()s with actual values.